spec/unit/plugin_manager/pack_installer/local_spec.rb

```diff
@@ -67,8 +67,8 @@
       it "install the gems" do
         expect(::Bundler::LogstashInjector).to receive(:inject!).with(be_kind_of(LogStash::PluginManager::PackInstaller::Pack)).and_return([])
 
-        expect(::LogStash::PluginManager::GemInstaller).to receive(:install).with(/logstash-input-packtest/, anything)
-        expect(::LogStash::PluginManager::GemInstaller).to receive(:install).with(/logstash-input-packtestdep/, anything)
+        expect(::LogStash::PluginManager::GemInstaller).to receive(:install).with(/logstash-input-packtest-/, anything)
```

Comment:
why is this change necessary?

Reply:
The commit that introduces the changes is 0c92eb34055188e7dd543758b58be5128ddba2a3.

Without this change, the test fail:
```sh
SPEC_OPTS="-fd -P spec/unit/plugin_manager/pack_installer/local_spec.rb" ./gradlew clean :logstash-core:rubyTests --tests org.logstash.RSpecTests
```

with 

```
    Failures:

      1) LogStash::PluginManager::PackInstaller::Local when the local file exist when the pack is valid install the gems
         Failure/Error: expect(::LogStash::PluginManager::GemInstaller).to receive(:install).with(/logstash-input-packtest/, anything)

           (LogStash::PluginManager::GemInstaller (class)).install(/logstash-input-packtest/, anything)
               expected: 1 time with arguments: (/logstash-input-packtest/, anything)
               received: 2 times with arguments: (/logstash-input-packtest/, anything)
         # ./spec/unit/plugin_manager/pack_installer/local_spec.rb:70:in `block in <main>'
         # ./lib/bootstrap/rspec.rb:36:in `<main>'

    Finished in 0.04501 seconds (files took 0.02848 seconds to load)
    5 examples, 1 failure

    Failed examples:

    rspec ./spec/unit/plugin_manager/pack_installer/local_spec.rb:67 # LogStash::PluginManager::PackInstaller::Local when the local file exist when the pack is valid install the gems


org.logstash.RSpecTests > rspecTests[core tests] FAILED
    java.lang.AssertionError: RSpec test suite `core tests` saw at least one failure.
        at org.junit.Assert.fail(Assert.java:88)
        at org.logstash.RSpecTests.rspecTests(RSpecTests.java:72)
```

Reply:
really strange that it worked before and now doesn't. main and this PR are using the same version of rspec, but bundler and jruby are different.